libs/designer/src/lib/core/utils/__test__/segment.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { beforeAll, describe, expect, it } from 'vitest';
+import { createLiteralValueSegment, ValueSegmentConvertor, ValueSegmentConvertorOptions } from '../parameters/segment';
+import constants from '../../../common/constants';
+
+describe('convertToValueSegments', () => {
+  const convertor = new ValueSegmentConvertor({
```

Comment:
Is this convertor only used in conditions? I am worried this might change other fields